packages/v4/patternfly-docs/content/design-guidelines/components/form-control/form-control.md

```diff
@@ -46,11 +46,15 @@ Use the read only state when an input will *never* be editable. This state diffe
 
 <img src="./img/greyedout.png" alt="Gray-out input" width="644px"/>
 
+#### Red delete text styling
+* Red delete text styling should only be used for very destructive actions.
+* A divider should be used for destructive actions.
+
+<img src="./img/red-delete-text.png" alt="red text down down menu" width="863”/>
```

Comment:
forgot the "px" at the end of the width. not sure if that makes a difference but worth a try